client/src/pages/Payments.js

```diff
@@ -36,38 +34,27 @@ var Payments = function () {
     return (
         <div>
             <Header />
-            <Grid container justify="center">
-                <Grid item xs={9} md={7} lg={12} className={classes.root}>
-                    <Paper className={classes.nav}>
-                        <BottomNavigation
-                            value={stripePage}
-                            onChange={(event, newValue) => {
-                                setStripePage(newValue);
-                            }}
-                            showLabels
-                            className={classes.root}
-                        >
-                            <BottomNavigationAction value='summary' label="Summary" icon={<LocationOn />} />
-                            <BottomNavigationAction value='creditCard' label="Credit Card" icon={<LocationOn />} />
-                            <BottomNavigationAction value='transfer' label="Bank Account" icon={<LocationOn />} />
-                            <BottomNavigationAction value='history' label="View Payment/Transfer History" icon={<LocationOn />} />
-                        </BottomNavigation>
-                    </Paper>
+            <Grid container component="main" className={classes.root}>
+                <CssBaseline />
+                <Grid item xs={3} sm={3} md={3} >
+                    <List><ListPaymentOptions setStripePage={setStripePage} /></List>
                 </Grid>
-                <Grid item xs={9} md={7} lg={12} className={classes.info}>
-                    {stripePage === 'creditCard' &&
-                        <StripeProvider apiKey="pk_test_3Ty6VUy1rfVdHm4JSOP1Uo8z00w8r5ooyx">
-                            <Elements>
-                                <InjectedCreditCard userID={userID} />
-                            </Elements>
-                        </StripeProvider>
-                    }
-                    {stripePage === 'transfer' && <Transfer userID={userID} />}
-                    {stripePage === 'history' && <History userID={userID} />}
-                    {stripePage === 'summary' && <h3>Account Summary</h3>}
+                <Grid item xs={9} sm={9} md={9} component={Paper} elevation={6} square>
+                    <div className={classes.paper}>
+                        {stripePage === 'creditCard' &&
+                            <StripeProvider apiKey="pk_test_3Ty6VUy1rfVdHm4JSOP1Uo8z00w8r5ooyx">
```

Comment:
hide apiKey

Reply:
For the client side, where would I hide the key? Should I request it from the server?  Even though it is the public key? Thanks